### ISSUE TITLE
Add 'Scrollable pane' component to MoJ Figma Kit

### ISSUE THREAD

**Issue by murrlipp** [ministryofjustice/moj-frontend#822]
## What
Add a component to the MoJ Figma Kit for the [Scrollable pane](https://design-patterns.service.justice.gov.uk/components/scrollable-pane/) component of the MoJ design system.

## Why
There is no Figma component for this and it is likely to help designers work more quickly.

## Who needs to be involved
- Interaction Designer

## Tasks
- [ ] Check playground for existing versions
- [ ] Create component
- [ ] Fill out release notes
- [ ] Publish library changes with release notes

**Comment by murrlipp** [comment]
@robertjmccarthy I don't think there's any value to adding this to the Figma library, for the following reasons:

1. We'd need to add infinite nested component instances and allow users to replace those instances with other components
2. We wouldn't be able to configure this to allow custom content (i.e. non-component items).
3. We wouldn't be able to accurately represent the scroll bar.

Are we happy for this not to appear in the Figma library?

**Comment by robertjmccarthy** [comment]
@murrlipp agree. Without those things it seems pointless adding it. Thanks for looking at it.
